<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouchWidget</class>
 <widget class="QWidget" name="TouchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Touch Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>8</number>
        </property>
        <item>
         <widget class="QCheckBox" name="touchEventsCheckBox">
          <property name="toolTip">
           <string>Turns on/off Touch events (default is enable/on).</string>
          </property>
          <property name="text">
           <string>Enable Touch events</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="scrollDirection">
          <property name="text">
           <string>Invert Scrolldirection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_Absolute">
          <property name="toolTip">
           <string>The mouse cursor will move exactly to the position on the tablet.</string>
          </property>
          <property name="text">
           <string comment="absolute pen movement (pen mode)">Absolute (Pen Mode)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_Relative">
          <property name="toolTip">
           <string>The mouse cursor can be moved freely. The tablet working space does not represent the screen space
 The speed of the mouse is the same as specified for any other mouse pointer. This can be changed in the mouse setting section</string>
          </property>
          <property name="text">
           <string comment="relative pen movement (mouse mode)">Relative (Mouse Mode)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Multi-Touch</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>8</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="gesturesCheckBox">
          <property name="toolTip">
           <string>Turns on/off multi-touch gesture events (default is enable/on). </string>
          </property>
          <property name="text">
           <string>Enable Gestures</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>Minimum distance for a zoom gesture (default is 50).</string>
          </property>
          <property name="text">
           <string>Zoom Distance:</string>
          </property>
          <property name="buddy">
           <cstring>zoomDistanceBox</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KIntSpinBox" name="zoomDistanceBox">
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>Minimum motion before sending a scroll gesture (default is 20).</string>
          </property>
          <property name="text">
           <string>Scroll Distance:</string>
          </property>
          <property name="buddy">
           <cstring>scrollDistanceBox</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KIntSpinBox" name="scrollDistanceBox">
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="toolTip">
           <string>Minimum time between taps for a right click (default is 250)</string>
          </property>
          <property name="text">
           <string>Tap time:</string>
          </property>
          <property name="buddy">
           <cstring>tapTimeBox</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="KIntSpinBox" name="tapTimeBox">
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>touchEventsCheckBox</tabstop>
  <tabstop>scrollDirection</tabstop>
  <tabstop>radioButton_Absolute</tabstop>
  <tabstop>radioButton_Relative</tabstop>
  <tabstop>gesturesCheckBox</tabstop>
  <tabstop>zoomDistanceBox</tabstop>
  <tabstop>scrollDistanceBox</tabstop>
  <tabstop>tapTimeBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>touchEventsCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>TouchWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gesturesCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>TouchWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomDistanceBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TouchWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrollDistanceBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TouchWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tapTimeBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TouchWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scrollDirection</sender>
   <signal>toggled(bool)</signal>
   <receiver>TouchWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Absolute</sender>
   <signal>toggled(bool)</signal>
   <receiver>TouchWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Relative</sender>
   <signal>toggled(bool)</signal>
   <receiver>TouchWidget</receiver>
   <slot>profileChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>profileChanged()</slot>
 </slots>
</ui>
